components/track/TrackMenu.vue

```diff
@@ -62,7 +64,20 @@ const dropdownMenuItemsForTrack = (track: TrackModel) => {
   items.push({
     icon: "icon.person",
     text: t("track.dropdown.go-to-contributors"),
-    link: { name: "browse" }, // TODO: change link
+    clickFunction: () => {
+      if (track.contributors && track.contributors?.length > 1) {
```

Comment:
Optional chaining shouldn't be needed when you've checked its existence. Since you're using optional chaining, you just need one check. The second condition could also be simplified by using the optional chaining on the array index.

Reply:
done